### ISSUE TITLE
0.7.29beta1 setcap is not static linked

### ISSUE THREAD

**Issue by naY9yjoS6ZqhOd35sIFH** [bedrocklinux/bedrocklinux-userland#295]
line 2307: /bedrock/libexec/setcap: No such file or directory
setcap elf seems dynamic
version 0.7.29beta1 
platform x86_64

**Comment by paradigm** [comment]
Good catch.  I can confirm the issue.  I double checked the other binaries and it didn't seem to hit them, just `setcap`.

There were two problems:

1. I updated the `setcap` version, and apparently it's build system has since introduced a bug where it no longer respects a request to statically link its binaries.[I found a work around.](https://github.com/bedrocklinux/bedrocklinux-userland/commit/72f235d6f383b9a541a11dd826319aa56cb93139)

2. While Bedrock's build system does explicitly check for this concern, apparently the check isn't sufficiently robust and this slipped through.  [I've improved the check accordingly.](https://github.com/bedrocklinux/bedrocklinux-userland/commit/cb13137fcb92183acb0d76a93d56fd7ffa4aaeb1)

0.7beta2 with these changes building now. I'll push the update once it's ready.
